util/os/windows.zig

```diff
@@ -0,0 +1,54 @@
+const std = @import("std");
+const log = std.log.scoped(.zbench_platform_windows);
+
+pub fn getCpuName(allocator: std.mem.Allocator) ![]const u8 {
+    const stdout = try exec(allocator, &.{ "wmic", "cpu", "get", "name" });
+    return stdout[45 .. stdout.len - 7];
```

Comment:
Might want to add a check here for sufficient length so that the slicing doesn't fail (multiple occurrences)

Reply:
True, missed that spot. Added.